Core/src/org/sleuthkit/autopsy/modules/exif/ExifParserFileIngestModule.java

```diff
@@ -65,8 +66,12 @@
     private long jobId;
     private static final IngestModuleReferenceCounter refCounter = new IngestModuleReferenceCounter();
     private FileTypeDetector fileTypeDetector;
+    private final HashSet supportedMimeTypes = new HashSet();
```

Comment:
sid, can you add the type parameter, so this doesn't generate compiler warnings.